srv/runner/runner.py

```diff
@@ -66,11 +73,18 @@ def run_job(self, job):
         Runs job
 
         """
+        # TODO: temporary solution, replace with the additional job in the pipeline?
+        # Download file and replace file URI param with local path
+        s3_uri = job.input_parameters[0]
+        file_path = self.__s3.copy_file_to_local(s3_uri, os.path.join(INPUT_PATH, job.id))
+        job.input_parameters = [file_path, os.path.join(OUTPUT_PATH, job.id)]
+
```

Comment:
It might be better if INPUT_PATH and OUTPUT_PATH gets from environment variables